package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Cumulus Framework for ingesting and processing NASA Earth data streams",
   "scripts": {
     "audit": "audit-ci --config ./audit-ci.json && lerna exec -- audit-ci --config $(pwd)/audit-ci.json",
+    "docs-build-packages": "for d in packages/*; do cd $d && npm run build-docs && cd -; done;",
```

Comment:
```suggestion
    "docs-build-packages": "lerna run build-docs",
```

Reply:
If we use `lerna run build-docs`, it will run it in the packages that have that command, and not run them in ones that don't. That way we won't need all of the `"build-docs": "true"` lines below.